johnsonandjohnson/Bodiless-JS
# fea(layouts) comp selector remember

### changes:
- Remember user's last filter selection in component library.

### Test Instruction:
-Use component library to filter components.
- Close the dialog.
- Reopen the dialog and notice the selection is remembered.

Fixes #653
